packages/web-components/package.json

```diff
@@ -52,8 +52,5 @@
     "puppeteer": "10.0.0",
     "react-dom": "^16.7.0",
     "typescript": "^4.3.5"
-  },
-  "peerDependencies": {
-    "@department-of-veterans-affairs/formation": "^6.14.0"
```

Comment:
I think we still want to list `formation` as a peer dependency since some of our components rely on those fonts and global styles in order to look right.

Reply:
Yeah, things will still look fine because [storybook is building with `formation`](https://github.com/department-of-veterans-affairs/component-library/blob/6e3b36cccb5d6745acb2705d4b67366abdcc35df/packages/storybook/.storybook/preview.js#L1-L3). If I remove `formation` from storybook yarn warns me about it:

![peer dependency warning in storybook](https://user-images.githubusercontent.com/2008881/163455122-01ecee91-8fae-4f20-8e5b-99392e5c8381.png)

And if I actually run storybook without `formation`, it doesn't look right:

![image](https://user-images.githubusercontent.com/2008881/163455563-21fd67eb-69d6-4da7-ab69-9571a41ddc85.png)

If I remove `formation` from `vets-website` I get a similar warning:

![peer dependency warning in vets-website](https://user-images.githubusercontent.com/2008881/163454365-c0a1360f-f276-454c-b659-4289954bb04c.png)

So having the peer dependency in place will give us warnings when they aren't met and can be useful for debugging if something isn't working as expected.

Reply:
@bkjohnson thanks for the info that makes a lot of sense I appreciate it. I added back in the peerDependency and bumped the version to the latest to keep everything in line.